### ISSUE TITLE
error in query with random

### ISSUE THREAD

**Issue by cocodrino** [alaisi/postgres.async#10]
Hi, this code

```clojure
(!/<!! (<query!  db ["SELECT * FROM articles ORDER BY random() LIMIT 1"]))
```

give me this exception

=> #<IllegalArgumentException java.lang.IllegalArgumentException: No method in multimethod 'from-pg-value' for dispatch value: BOOL>

the query is correct (I just try it in the postgre and works) but seems than postgres.async cant interprete it well.

**Comment by alaisi** [comment]
Thanks for the report.

The problem is not with random, but with boolean type. I'll submit a fix to the Java driver.

**Comment by alaisi** [comment]
Tracked issue: https://github.com/alaisi/postgres-async-driver/issues/5

**Comment by alaisi** [comment]
Fixed in release 0.5.0

**Comment by cocodrino** [comment]
thanks :+1:
